draft-ietf-acme-acme.md

```diff
@@ -2512,7 +2512,7 @@ Initial Contents
 
 | Label      | Identifier Type | Reference |
 |:-----------|:----------------|:----------|
-| http-01    | dns             | RFC XXXX  |
+| http-07    | dns             | RFC XXXX  |
```

Comment:
It doesn't seem like the table still contains `tls-sni-01`. Is there some reason to include `http-01`, but not `tls-sni-01`?